http2_general/test_h2_sticky_cookie.py

```diff
@@ -0,0 +1,185 @@
+"""Test module for http2 and Sticky Cookie."""
+from framework import tester
+
+
+class H2StickyCookieBaseTestCase(tester.TempestaTest):
```

Comment:
If this is the base class to be inherited from, we should use it with `(tester.TempestaTest, base=True)`. This way no tests will run for this particular class. Usually you'd want to do it when you have many similar test cases and only some minor part differs.